### PR TITLE
Added path/command configurability to support custom tooling

### PR DESCRIPTION
**Pull request by hoovercj**
Defaults to p4, but can be configured to run a different command to support p4 forks or wrappers

### CODE REVIEW

File: perforce.js [hunk 1]
```diff
@@ -51,15 +56,15 @@ function activate() {
...
 	}
 }
 exports.activate = activate;
 
 function buildCmdline(command, args)
 {
-	var cmdline = "p4";
+	var cmdline = _cmdline;
 	if (isWin) {
 		cmdline += ".exe";
```
Comment (stef-levesque) [inline perforce.js]: Hm, appending '.exe' for Windows will be prone to error. Can we have a 'per-plaform' default in the configuration ?

Comment (hoovercj) [inline perforce.js]: That part was already there. I agree that there might be a better solution but I think it's outside the scope of this PR

Comment (stef-levesque) [inline perforce.js]: I'll look into it later, thanks for the feature :+1:

**Comment by hoovercj** [comment]
Great, thanks. Please let me know if/when when you publish this to the marketplace.

**Comment by stef-levesque** [comment]
I'll test it against the latest VSCode release and probably publish it tonight.

**Comment by stef-levesque** [comment]
Version 0.1.7 is now available at https://marketplace.visualstudio.com/items?itemName=slevesque.perforce

**Comment by hoovercj** [comment]
Great, thanks!

**Pull request merged**